accepted/2021/system-drawing-win-only/system-drawing-win-only.md

```diff
@@ -0,0 +1,193 @@
+
+Thus, we believe it is not worth investing in making `System.Drawing.Common` work
+on non-Windows platforms and instead propose we only keep evolving it in the
+context of Windows Forms and GDI+.
+
+## Requirements
+
+### Goals
+
+* Consumers of `System.Drawing.Common` will be informed that starting with .NET
+  6, it's only supported on Windows, via the platform compatibility analyzer
+* Cross-platform consumers can reference older versions of the package to
+  continue using `System.Drawing.Common` via `libgdiplus`
+
+### Non-Goals
+
+* Closing the gap of `System.Drawing.Common` for non-Windows platforms or
+  bundling `libgdiplus` as part of the .NET 6 installer
+
+## Stakeholders and Reviewers
+
+* Libraries team
+* ML.NET
+* Xamarin team
+
+## Design
+
+In the past we decided to include the following disclaimer in our docs:
+
+>   The `System.Drawing.Common` NuGet package works on Windows, Linux, and macOS.
+>   However, there are some platform differences. On Linux and macOS, the GDI+
+>   functionality is implemented by the `libgdiplus` library. This library is not
+>   installed by default in most Linux distributions and doesn't support all the
+>   functionality of GDI+ on Windows on macOS. There are also platforms where
+>   `libgdiplus` is not available at all. To use types from the
+>   `System.Drawing.Common` package on Linux and macOS, you must install
+>   `libgdiplus` separately. For more information, see Install .NET on Linux or
+>   Install .NET on macOS.
+>
+>   If you can't use System.Drawing with your application, recommended
+>   alternatives include ImageSharp, SkiaSharp, and Windows Imaging Components.
+
+However, this disclaimer doesn't feel clear enough to steer people towards using
+other graphics libraries.
+
+The proposal is to do the following for .NET 6:
+
+1. **Mark the assembly as only being supported on Windows**. That will cause
+   the platform compatibility analyzer to warn when consumed from a project
+   that isn't targeting net6.0-windows (or marked as Windows-specific).
+
+2. Throw `PlatformNotSupportedException` when trying to load `libgdiplus`
+   pointing to an aka.ms link with the reasons behind it and recommending
+   stable, well-maintained open-source library alternatives
+
+3. Add targets to the package that when it is restored using a non-windows RID,
+   it warns that we no longer support it on non-windows and point to the aka.ms
+   link with more info.
+
+The challenge is that `System.Drawing.Common` is used by 1st party consumers
+such as ML.NET, which uses it for image manipulation and even exposes an API
+that returns a Bitmap. At the same time, https://github.com/dotnet/iot is in the process
+of moving to use ImageSharp and so far they had a great migration experience. We
+need to engage with 1st party customers to see what impact this proposal will
+have on them.
+
+***Note:** This only affect `System.Drawing.Common` and would leave
```

Comment:
```suggestion
***Note:*** This only affect `System.Drawing.Common` and would leave
```
The rest of the doc is rendered in bold for some reason. Is this going to fix it?

Reply:
There was a missing `*` at the end of the paragraph (in Markdown, `*` and `_` can span multiple lines). The intention is that this entire paragraph is italicized while the first word is bolded. This is how it renders here: 

![image](https://user-images.githubusercontent.com/5169960/125866879-f7b54d53-3752-4951-bdd8-71fc4333888f.png)

Reply:
It still does not render correctly in the markdown with syntax highlighting view:

![image](https://user-images.githubusercontent.com/6668460/125867070-18f70771-6455-4ee4-bac3-3c5f9c407747.png)

Reply:
Yeah, the syntax highlighter uses some weird non-compliant Markdown interpretation 🤷